From: Alban Peignier <alban@tryphon.eu>
Date: Sat, 27 Aug 2011 15:33:28 +0200
Subject: Remove debian directories from Makefile list

---
 configure.in                                       |    2 --
 1 file changed, 2 deletions(-)

diff --git a/configure.in b/configure.in
index 2695911..dd0063d 100644
--- a/configure.in
+++ b/configure.in
@@ -383,8 +383,6 @@ AC_CONFIG_FILES([rivendell.spec \
     docs/Makefile \
     docs/tables/Makefile \
     docs/examples/Makefile \
-    debian/Makefile \
-    debian/patches/Makefile \
     xdg/Makefile \
     rdadmin/Makefile \
     scripts/Makefile \
