From: Alban Peignier <alban@tryphon.eu>
Date: Sat, 27 Aug 2011 15:33:28 +0200
Subject: Remove debian directories from Makefile list

---
 configure    |    4 +---
 configure.in |    2 --
 2 files changed, 1 insertions(+), 5 deletions(-)

diff --git a/configure b/configure
index 419d02f..72f8010 100755
--- a/configure.in
+++ b/configure.in
@@ -385,8 +385,6 @@ AC_CONFIG_FILES([rivendell.spec \
     docs/Makefile \
     docs/tables/Makefile \
     docs/examples/Makefile \
-    debian/Makefile \
-    debian/patches/Makefile \
     xdg/Makefile \
     rdadmin/Makefile \
     scripts/Makefile \
