From: Alban Peignier <alban@tryphon.eu>
Date: Sun, 28 Aug 2011 10:49:48 +0200
Subject: Fix headerdir in rlm makefile

---
 rlm/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rlm/Makefile.am b/rlm/Makefile.am
index f9eeb33..22309da 100644
--- a/rlm/Makefile.am
+++ b/rlm/Makefile.am
@@ -54,7 +54,7 @@ install:	all
 	cp *.c $(DESTDIR)$(prefix)/src/rlm/
 	cp Makefile-example $(DESTDIR)$(prefix)/src/rlm/Makefile
 
-headerdir = $(includedir)/rlm
+headerdir = $(DESTDIR)$(includedir)/rlm
 header_HEADERS = rlm.h
 
 EXTRA_DIST = Makefile-example\
