From: Alban Peignier <alban@tryphon.eu>
Date: Sun, 28 Aug 2011 10:49:48 +0200
Subject: Fix headerdir in rlm makefile

---
 rlm/Makefile.am |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rlm/Makefile.am b/rlm/Makefile.am
index 5008d3b..3e0cb0e 100644
--- a/rlm/Makefile.am
+++ b/rlm/Makefile.am
@@ -50,7 +50,7 @@ install:	all
 	mkdir -p $(headerdir)
 	cp rlm.h $(headerdir)
 
-headerdir = $(includedir)/rlm
+headerdir = $(DESTDIR)$(includedir)/rlm
 header_HEADERS = rlm.h
 
 EXTRA_DIST = Makefile-example\
