From: Alban Peignier <alban@tryphon.eu>
Date: Sun, 28 Aug 2011 10:16:06 +0200
Subject: Remove debian directory from Makefile.am

---
 Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 581282a..4d6976a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -32,7 +32,7 @@ if ALSA_RD_AM
   ALSACONFIG_RD_OPT = rdalsaconfig
 endif
 SUBDIRS = icons helpers $(HPI_RD_OPT) lib rlm scripts cae conf docs \
-          debian xdg rdadmin ripcd rdlogin rdlibrary rdcatch rdcatchd \
+          xdg rdadmin ripcd rdlogin rdlibrary rdcatch rdcatchd \
           rdlogedit rdlogmanager rdpanel rdairplay rdcastmanager rdrepld \
           utils web tests importers ios $(PAM_RD_OPT)
 
-- 
