From: Alban Peignier <alban@tryphon.eu>
Date: Sat, 15 Oct 2011 16:55:37 +0200
Subject: Replace cdda2wav by icedax

---
 lib/rdcddblookup.cpp |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/lib/rdcddblookup.cpp b/lib/rdcddblookup.cpp
index 442b2ed..dd76b44 100644
--- a/lib/rdcddblookup.cpp
+++ b/lib/rdcddblookup.cpp
@@ -330,11 +330,11 @@ bool RDCddbLookup::GetCddaData(QString &cdda_dir,QString &cdda_dev)
   //
   // Write the Track Title Data to a Temp File
   //
-  QString cmd=QString().sprintf("CURDIR=`pwd`;cd %s;cdda2wav -D %s --info-only -v titles 2> /dev/null;cd $CURDIR",
+  QString cmd=QString().sprintf("CURDIR=`pwd`;cd %s;icedax -D %s --info-only -v titles 2> /dev/null;cd $CURDIR",
 				(const char *)cdda_dir,
 				(const char *)cdda_dev);
   if((err=system(cmd))!=0) {
-    printf("cdda2wav error: %d\n",err);
+    printf("icedax error: %d\n",err);
     return false;
   }
 
@@ -376,11 +376,11 @@ bool RDCddbLookup::GetCddaData(QString &cdda_dir,QString &cdda_dev)
   //
   // Write the ISRC Data to a Temp File
   //
-  cmd=QString().sprintf("CURDIR=`pwd`;cd %s;cdda2wav -D %s --info-only -v trackid 2> /dev/null;cd $CURDIR",
+  cmd=QString().sprintf("CURDIR=`pwd`;cd %s;icedax -D %s --info-only -v trackid 2> /dev/null;cd $CURDIR",
 				(const char *)cdda_dir,
 				(const char *)cdda_dev);
   if((err=system(cmd))!=0) {
-    printf("cdda2wav error: %d\n",err);
+    printf("icedax error: %d\n",err);
     return false;
   }
 
