From: Alban Peignier <alban@tryphon.eu>
Date: Sun, 28 Aug 2011 16:21:59 +0200
Subject: No lib64 for rd_lib_path

---
 configure.in |    7 ++-----
 1 files changed, 2 insertions(+), 5 deletions(-)

diff --git a/configure.in b/configure.in
index ac562e1..4c79499 100644
--- a/configure.in
+++ b/configure.in
@@ -341,11 +341,8 @@ else
   AC_SUBST(RPM_ROOT,/usr/src/packages)
   AC_SUBST(VENDOR,suse)
 fi 
-if test $ar_gcc_arch = x86_64 ; then
-  AC_SUBST(RD_LIB_PATH,lib64)
-else
-  AC_SUBST(RD_LIB_PATH,lib)
-fi
+# Always use lib debian/ubuntu
+AC_SUBST(RD_LIB_PATH,lib)
 
 AC_CONFIG_FILES([rivendell.spec \
     Makefile \
