From: Alban Peignier <alban@tryphon.eu>
Date: Sun, 24 Mar 2013 10:33:03 +0100
Subject: Remove debian directory from Makefile.am

---
 Makefile.am | 1 -
 1 file changed, 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index 4fc4bf0..3bbad49 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -37,7 +37,6 @@ SUBDIRS = icons\
           $(HPI_RD_OPT) $(PAM_RD_OPT) rlm\
           scripts\
           conf\
-          debian\
           docs\
           xdg\
           cae\
