From: Alban Peignier <alban@tryphon.eu>
Date: Wed, 22 May 2013 19:57:40 +0200
Subject: No lib64 for rd_lib_path

---
 configure.ac | 10 +++-------
 1 file changed, 3 insertions(+), 7 deletions(-)

diff --git a/configure.ac b/configure.ac
index bda3fa1..c21ed6d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -93,13 +93,9 @@ fi
 AR_GCC_TARGET()
 AC_SUBST(VENDOR,$ar_gcc_distro)
 AC_SUBST(ARCH,$ar_gcc_arch)
-if test $ar_gcc_arch = x86_64 ; then
-  LIB_PATH=$PREFIX/lib64
-  AC_SUBST(RD_LIB_PATH,lib64)
-else
-  LIB_PATH=$PREFIX/lib
-  AC_SUBST(RD_LIB_PATH,lib)
-fi
+
+# Always use lib debian/ubuntu
+AC_SUBST(RD_LIB_PATH,lib)
 
 #
 # Determine Distro
